### ISSUE TITLE
[5.6] Collection toArray() method return array of objects instead of an array

### ISSUE THREAD

**Issue by StAngerGordun** [laravel/framework#25235]
- Laravel Version: 5.6.33
- PHP Version: 7.2
- Database Driver & Version: MariaDB 10.2

### Description:
Method `toArray()` array of objects instead of an array. 

### Steps To Reproduce:
 This code  does not return a full array:
`$resultArray = DB::table('table_name')
            ->get()->toArray();`

To fully convert the result to an array, you need to use the following code:
`$resultArray  = array_map(
            function($value) {
                return (array)$value;
            }, $resultArray 
        );`

**Comment by X-Coder264** [comment]
The `get()` method returns a `Collection` instance (which is just a nice wrapper around PHP arrays) which in this case has `stdClass` objects inside since `PDO::FETCH_OBJ` is used by default. The `toArray()` method call on the collection instance gives you the plain PHP array of items that the collection had. That array is of course consisted of `stdClass` objects.

TL;DR This is not a bug.
